@here/harp-omv-datasource/lib/IGeometryProcessor.ts

```diff
@@ -30,7 +14,7 @@ export interface IPolygonGeometry {
     /**
      * The rings of this polygon.
```

Comment:
Please add a comment in which space the points are

Reply:
Done.